include/mbedtls/ssl.h

```diff
@@ -1118,10 +1118,11 @@ struct mbedtls_ssl_session
 
     unsigned char MBEDTLS_PRIVATE(exported);
 
-    /* This field is temporarily duplicated with mbedtls_ssl_context.minor_ver.
-     * Once runtime negotiation of TLS 1.2 and TLS 1.3 is implemented, it needs
-     * to be studied whether one of them can be removed. */
-    unsigned char MBEDTLS_PRIVATE(minor_ver);    /*!< The TLS version used in the session. */
+    /*!< Minor version negotiated in the session. Used if and when
```

Comment:
Doxygen issue: I think `/*!<` is used when the documentation follows the thing, but when it's before it, that should be `/**` (or `/*!` but less common in our codebase I think). https://www.doxygen.nl/manual/docblocks.html

(Otherwise, thanks a lot for the new documentation, it's really clear now.)

Reply:
ok I will address in a follow-up PR. Thanks for the quick reviews.